backend/tests/test_integration_org_unit.py

```diff
@@ -33,6 +33,9 @@ def test_org_unit_temporality(self):
                     'location': ('Overordnet Enhed/Humanistisk fakultet/' +
                                  'Historisk Institut'),
                     "user_key": "frem",
+                    'user_settings': {'orgunit': {'show_bvn': False,
+                                                  'show_location': True,
+                                                  'show_roles': True}},
```

Comment:
I'm not fond of including this with each and every request — it seems a bit too specific to the UI. Considering that we don't expect the configuration to change, we should consider just exposing it, and let the frontend determine whether to show anything. As it happens, you recently made the change to give it the entire tree, right?